docs/agents/hwp_supervisor_agent.rst

```diff
@@ -38,9 +38,32 @@ Here is an example of a config block you can add to your ocs site-config file::
             '--ups-id', 'power-ups-az',
             '--ups-minutes-remaining-thresh', 45,
             '--iboot-id', 'power-iboot-hwp-2',
-            '--iboot-outlets', [1,2]
+            '--driver-iboot-id', 'hwp-synaccess-1',
+            '--driver-iboot-outlets', 4, 5,
```

Comment:
I do agree... but looking through, there are many places in the code / docs where "iboot" is used like this, both for the driver power and the gripper power options and vars. 

I personally think it would be more confusing to only partially fix this compared to leaving as is and maybe noting in the docs that iboot is shorthand for power-control device... I'd also rather not deprecate it for SATp1 if thats ok w/ you.

maybe we write this up as an issue and do in another pr?

Reply:
Yeah sounds good. Should have been caught when the option was introduced, but it stood out in the documentation to me.